content/reference/changelog.md

```diff
@@ -1,5 +1,11 @@
 # Changelog
 
+## 2017-10-02
+  * [api improvement] Add `add_bookings_fees` endpoint
+  * [api improvement] Add `remove_bookings_fees` endpoint
```

Comment:
Why duplicating? :P

Reply:
@Azdaroth Seemed like the convention, I can merge it if you'd like

Reply:
Wow, is it really duplicated with the same case for 2 different labels?

Reply:
@kaizencodes ^

Reply:
@Azdaroth done